pkg/util/mount/mount_linux_test.go

```diff
@@ -151,3 +151,32 @@ func setEquivalent(set1, set2 []string) bool {
 	}
 	return true
 }
+
+func TestGetDeviceNameFromMount(t *testing.T) {
+	fm := &FakeMounter{
+		MountPoints: []MountPoint{
+			{Device: "/dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2014-12.server:storage.target01-lun-1",
```

Comment:
I asked this before - this is a test, so the names can be shorter and more comprehensible.